src/Providers/ViewModelsServiceProvider.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace Spatie\ViewModels\Providers;
+
+use Illuminate\Support\ServiceProvider;
+use Spatie\ViewModels\Console\ViewModelMakeCommand;
+
+class ViewModelsServiceProvider extends ServiceProvider
+{
+    /**
+     * Bootstrap services.
+     *
+     * @return void
+     */
+    public function boot()
+    {
+        //
+    }
+
+    /**
+     * Register services.
+     *
+     * @return void
+     */
+    public function register()
+    {
+        $this->commands([
```

Comment:
Shouldn't this command be register only if the application is on console?

```php
        if ($this->app->runningInConsole()) {
            $this->commands([
                ViewModelMakeCommand::class,
            ]);
        }
```

Reply:
Yup, you're right! 

@ijpatricio 
![image](https://user-images.githubusercontent.com/483853/46004819-1db75300-c0b4-11e8-947c-94ce30a8f799.png)

Reply:
😃

Reply:
done